azext_edge/edge/_help.py

```diff
@@ -586,9 +587,14 @@ def load_iotops_help():
             data_source={data_source} name={name} observability_mode={observability_mode} sampling_interval={sampling_interval}
             queue_size={queue_size} --data data_source={data_source}
 
+        - name: Create an asset using a file containing data-points and another file containing events.
+          text: >
+            az iot ops asset create --name MyAsset -g MyRg --custom-location MyLocation --endpoint exampleEndpoint
+            --data-file /path/to/myasset_datapoints.json --event-file /path/to/myasset_events.csv
+
         - name: Create an asset with the given pre-filled values.
           text: >
-            az iot ops asset create --name MyAsset -g MyRg --custom-location MyLocation --endpoint example.com
+            az iot ops asset create --name MyAsset -g MyRg --custom-location MyLocation --endpoint exampleEndpoint
```

Comment:
Is --endpoint intended to be the endpoint profile name? If so --endpoint-profile is more clear.

Reply:
yes and future fix